scripts/report-bundle-size/index.js

```diff
@@ -0,0 +1,32 @@
+const path = require('path')
+const fs = require('fs')
+const { browserSdkVersion } = require('../lib/browser-sdk-version')
+const { runMain } = require('../lib/execution-utils')
+const { reportBundleSizes } = require('./report-as-a-pr-comment')
+const { createLogData, sendLogToOrg2 } = require('./report-to-datadog')
+
+const rumPath = path.join(__dirname, '../../packages/rum/bundle/datadog-rum.js')
+const logsPath = path.join(__dirname, '../../packages/logs/bundle/datadog-logs.js')
+const rumSlimPath = path.join(__dirname, '../../packages/rum-slim/bundle/datadog-rum-slim.js')
+const workerPath = path.join(__dirname, '../../packages/worker/bundle/worker.js')
+
+runMain(async () => {
+  const bundleSizes = {
+    rum: getBundleSize(rumPath),
+    logs: getBundleSize(logsPath),
+    rum_slim: getBundleSize(rumSlimPath),
+    worker: getBundleSize(workerPath),
+  }
+  const logData = createLogData(bundleSizes, browserSdkVersion)
+  await sendLogToOrg2(logData)
```

Comment:
**💬 suggestion:** to make the main script even more clearer, you could create a function like this in `report-to-datadog.ts`:

```js
const { browserSdkVersion } = require('../lib/browser-sdk-version')

async function reportBundleSizesToDatadog(bundleSizes) {
  const logData = createLogData(bundleSizes, browserSdkVersion)
  await sendLogToOrg2(logData)
}
```